projectsmember/signin.aspx.cs

```diff
@@ -31,8 +29,17 @@ protected void btnLogin_Click(object sender, EventArgs e)
 
             if (numberOfUsers > 0)
             {
+                var allChar = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+                var random = new Random();
+                var resultToken = new string(
+                   Enumerable.Repeat(allChar, 5)
+                   .Select(token => token[random.Next(token.Length)]).ToArray());
+
+                string authToken = resultToken.ToString();
+                DataSet1TableAdapters.MembersTableAdapter dstaToken = new DataSet1TableAdapters.MembersTableAdapter();
+                dstaToken.UpdateQuery(authToken, txtUsername.Text, txtPass.Text);
```

Comment:
do we really need the password here? Storing password in two different tables can add some complexities, what if the member changes the password?